src/api/cli/cli.go

```diff
@@ -147,7 +148,11 @@ func LoadConfig() (Config, error) {
 		return Config{}, ErrWalletName
 	}
 
-	dataDir := filepath.Join(home, fmt.Sprintf(".%s", coin))
+	// get data dir dir from env
+	dataDir := os.Getenv("DATA_DIR")
+	if dataDir == "" {
+		dataDir = filepath.Join(defaultDataDir)
```

Comment:
It was correct before. 

And this dataDir must be used to construct the wallet dir, if the wallet dir is not set.